docs/source/software/conda.rst

```diff
@@ -16,14 +19,15 @@ Using a locally installed Conda virtual environment is highly recommended so tha
-4. To create your environment, type ``conda create -n <yourenvironmentname> python=3.7 anaconda``, where <yourenvironmentname> is the name you want to give your environment. Tip: to see a list of all of your conda environments, type ``conda info -e``. To save space, also use the ``--prefix=/work/<mygroup>/<mydirectory>`` flag to build in your work directory.
+2. To load anaconda, type ``module load anaconda3/2022.01``.
+3. To check what version of Python you have installed, type ``which python``. With the anaconda module loaded you should see the python directory as /shared/centos7/anaconda3/2022.01/bin/python. If you wish to use a different version of python you can load a python module. For example, ``module load python/3.8.1`` and then confirm that your python directory has changed with ``which python``.
+4. To create your environment in the /work or /scratch directory, first create a project folder for the environment using ``mkdir <yourenvironmentname>``. And now create the environment with ``conda create --prefix=/work/<mygroup>/<yourenvironmentname>``. You will see a message "WARNING: A directory already exists at the target location '/work/<mygroup>/<yourenvironmentname>' but is not a conda environment. Enter y to continue creating the environment. Tip: to see a list of all of your Conda environments, type ``conda info -e``.
 5. Follow the prompts to complete the Conda install.
-6. To activate your Conda environment, type ``source activate <yourenvironmentname>``. Note that ``conda activate`` will not work on Discovery with this version.
-7. To install a specific package, type ``conda install -n <yourenvironmentname> [package]``.
-8. To deactivate the current, active Conda environment, type ``conda deactivate``.
-9. To delete a Conda environment and all of its related packages, type ``conda remove -n <yourenvironmentname> --all``.
+6. To activate your Conda environment, type ``conda activate <yourenvironmentname>/``. Note, this command will only work with the ``/`` at the end of your environment name. You should now see the environment name on the left side of your username and current node information: (/work/group/username/test_condaRTD) [username@c2000 dirname]$
+7. If you have a longer directory path within your /scratch or /work your command prompt can get over crowded. You can clear this up by running the command: ``conda config --set env_prompt '({name}) '`` which will modify your .condarc file to only show the active envrionments name and not its full path: (test_condaRTD) [username@c2000 dirname]$
+8. With the conda environment activated you can install a package within that environment with ``conda install [package]``. Alternatively, you can install a specific package from your default shell with ``conda install -n <yourenvironmentname> [package]``. This second approach runs the risk of unintentionally installing a package system-wide and we recommend installing packages from within the specific environment.
```

Comment:
Alternatively, you can install a specific package from your default shell with ``conda install -n <yourenvironmentname> [package]``. This second approach runs the risk of unintentionally installing a package system-wide and we recommend installing packages from within the specific environment.

I don't know if we should offer this in the RTD. We might get questions on it and it isn't too hard to just activate the environment you are trying to install packages into. 

Also users shouldn't be able to install system wide packages because they don't have write access to /shared/centos7/anaconda/... I didn't test it but the system should prevent that with write permissions.